web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -119,46 +121,37 @@ function ProjectDetails(): JSX.Element {
     ? projectByOnChainId?.data.projectByOnChainId
     : projectByHandle?.data.projectByHandle;
 
-  // Legacy projects use project.metadata. On-chain projects use IRI resolver.
-  const projectTableMetadata: ProjectMetadataLD = project?.metadata;
-  const iriResolvedMetadata = useQuery(
-    getMetadataQuery({ iri: onChainProject?.metadata }),
-  );
-  const metadata = iriResolvedMetadata.data ?? projectTableMetadata;
-
-  const managementActions =
-    metadata?.['regen:landManagementActions']?.['@list'];
+  /** Anchored project metadata comes from IRI resolver. */
```

Comment:
another nit; might be worth reviewing and updating some of the other comments in this file. as i have had a little trouble understanding some of them.

Reply:
which ones exactly?

Reply:
it was this block of code and the comments with it, that threw me off a little bit:

```
  // if projectId is handle, query project by handle
  const { data: projectByHandle, isInitialLoading: loadingProjectByHandle } =
    useQuery(
      getProjectByHandleQuery({
        client: graphqlClient,
        enabled: !!projectId && !isOnChainId,
        handle: projectId as string,
      }),
    );

  const projectByHandleOnChainId =
    projectByHandle?.data.projectByHandle?.onChainId ?? undefined;
  const onChainProjectId = isOnChainId ? projectId : projectByHandleOnChainId;
  // else fetch project by onChainId
  const {
    data: projectByOnChainId,
    isInitialLoading: loadingProjectByOnChainId,
  } = useQuery(
    getProjectByOnChainIdQuery({
      client: graphqlClient,
      enabled: !!onChainProjectId,
      onChainId: onChainProjectId as string,
    }),
  );
```

it mentions if/else logic, and there is no explicit if/else statement. after looking at the code long enough i could see why there is an if/else logic here via the `enabled` parameter. it is sort of hidden and just confused me a bit. my preference would be to have an explicit if/else block. but no strong opinions, it makes sense after looking at it for a bit

Reply:
you cannot use react hooks within if/else conditions, it's part of the rules: https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level (and [reasons behind it](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)), hence the use of `enabled` which is already largely used across the app for handling such queries

Reply:
thanks for explaining it :-)